File: apps/docs/docs/.vitepress/examples/line-chart.ts
const catchLocationsArray = Object.entries(lakeLocations).map(
  ([lake, { x, y }]) => ({ lake, x, y })
);

gf.Layer([
  gf
    .Chart(catchLocationsArray)
    .flow(gf.scatter({ by: "lake", x: "x", y: "y" }))
    .mark(gf.blank().name("points")),
  gf.Chart(gf.select("points")).mark(gf.line()),
]).render(root, {
  w: 500,
  h: 300,
  axes: true,
});

File: packages/gofish-python/stories/forwardsyntax/line_chart.py
"""Forward Syntax V3/Line Chart — mirrors LineChart.stories.tsx"""

from gofish import Layer, chart, scatter, blank, select, line
from stories.data.seafood import catch_locations_array

TITLE = "Forward Syntax V3/Line Chart"


def default(w=400, h=400):
    points = (
        chart(catch_locations_array)
        .flow(scatter("lake", x="x", y="y"))
        .mark(blank().name("points"))
    )
    lines = chart(select("points")).mark(line())
    return Layer([points, lines])
